app/views/citizens/additional_accounts/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for :navigation do %>
+  <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+  <%= form_tag(citizens_additional_accounts_path) do %>
+
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+          <h1 class="govuk-fieldset__heading">
+            Do you have accounts with other banks?
+          </h1>
+        </legend>
+        <br />
+        <span class="govuk-hint">
+          Share details of any accounts you have with another bank or building society
+        </span>
+        <% if flash[:alert] %>
+          <%= flash[:alert] %>
+        <% end %>
+        <div class="govuk-radios">
```

Comment:
💭 Given this is the first time we're defining a radio button with the new designs, it would be 👍 to try to abstract that similarly to [what was done to text fields](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/ap-71-add-accounts-for-more-than-one-bank/app/views/shared/forms/_text_fields.html.erb), so that going forward anyone could just define a new radio group by supplying the the field and the radio values allowed for it (similary to what Rails does by default). The idea behind this, again, would be to avoid having the exactly same HTML across multiple templates where we're effectively just changing the values for the radios, not the radios HTML itself. DRY.